### PR TITLE
feat(index.js): added labels for markdown and preview

### PR DESCRIPTION
**Pull request by arteevraina**
##  Fixes Issue #345  #337 
Separate labels for `Markdown` and `Preview` have been added.

## Screenshot
image

Please Review and suggest changes @irmerk @Michael-Grover @DianaLease  :)

### CODE REVIEW

File: examples/src/index.js [hunk 1]
```diff
@@ -140,7 +142,7 @@ function Demo() {
             />
           </Grid.Column>
         </Grid>
-        <Divider vertical>Preview</Divider>
+        <Divider vertical></Divider>
```
Comment (DianaLease) [inline examples/src/index.js:145]: This can be self-closing (`<Divider vertical />`

**Comment by irmerk** [comment]
I want to wait for #315 to be finished to see how that would affect this PR.

File: examples/src/index.js (hunk 1 above)
Comment (arteevraina) [reply examples/src/index.js]: Yes, made the change. Thanks :+1:

**Comment by arteevraina** [comment]
## With Shadow
image
## Without Shawdow
image

@irmerk @DianaLease @Michael-Grover I have added the shadow under formatting toolbar and changed the label size to 18px.

**Comment by irmerk** [comment]
You'll need to rebase. to resolve conflicts.

**Comment by arteevraina** [comment]
@irmerk Do I still need to `rebase` it if it does not have merge conflicts?

**Comment by irmerk** [comment]
Yeah there are still conflicts, and we don't want merge commits in the git log, so you'll need to interactive rebase and get rid of that.

**Comment by arteevraina** [comment]
Sure @irmerk I will do that :)

**Comment by arteevraina** [comment]
@irmerk I will close this PR and create a fresh PR for this.

**Pull request closed without merging**